rllib/agents/cql/tests/test_cql.py

```diff
@@ -42,22 +42,28 @@ def test_cql_compilation(self):
 
         config["num_workers"] = 0  # Run locally.
         config["twin_q"] = True
-        config["clip_actions"] = False
+        config["clip_actions"] = True
```

Comment:
In case the offline data contains unclipped actions (e.g. from a PPO pre-run), we should set this to True, just in case. Otherwise, it could mess up the BC phase.